README.md

```diff
@@ -1,2 +1,179 @@
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="nameLogo.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello world Project <a name="about-project"></a>
+
+> Solo Microverse project, setting up a "Hello Microverse" project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1- HTML.
+2- CSS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML & CSS.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file.
+- Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Web browser.
+2. Code editor.
+3. Git-smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+    cd my-folder
+    git clone https://github.com/PabloEGonz/Hello-world.git
+```
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+    cd my-project
+    npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extention.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 Pablo Gonzalez
+
+- GitHub: [@PabloEGonz](https://github.com/PabloEGonz)
+- Twitter: [@PabloEzGon](https://twitter.com/PabloEzGon)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/pablo-ezequiel-gonz%C3%A1lez-ramos-b9b854265)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ]  Kindly remove all the comments under the sections in the readme file. These comments are just to guide you on how to fill the readme file correctly.